resources/quality/hms434/abs-fr/hms434_0.4_absfr_high.inst.cfg

```diff
@@ -0,0 +1,37 @@
+[general]
+version = 4
+name = High
+definition = hms434
+
+[metadata]
+setting_version = 8
+type = quality
+quality_type = high
+weight = 0
```

Comment:
This weight is used for sorting. I would use `1` here

Reply:
In a later commit, -1 is used for coarse, 0 for normal and 1 for high. These have been used consistently so this comment can be closed now